examples/dummy/asyncapi.yaml

```diff
@@ -4,6 +4,9 @@ info:
   version: 1.0.0
   description: This service is in charge of processing user signups
 servers:
+  mykafka:
+    url: kafka://localhost:9092
+    protocol: kafka
```

Comment:
If you're gonna use Confluent instead of a local Kafka broker, this URL should be pointing to your Confluent Kafka broker and the protocol should be `kafka-secure`. You don't have to push the code changes but that's how you should be testing it locally.